models/User.js

```diff
@@ -1,38 +1,45 @@
-const { Sequelize, DataTypes } = require('sequelize');
+const { Sequelize, Op, Model, DataTypes } = require('sequelize');
 const bcrypt = require('bcrypt');
 
 const sequelize = new Sequelize('hamishgpt', 'root', 'root', {

```

Comment:
@rrarima @wendy-holberton I don't know who is in charge of this, but please do not write in clear your database information (url, user, and credentials) like this and push on GitHub, even if your repo is private.

You have to write this information in a `.env` file never push on GitHub, like in the Front end.

Reply:
I set up a .env file with a SECERT_KEY

Reply:
Let me know when you've done it @wendy-holberton on hamishgpt slack and I'll have a look at it.,

Reply:
I've updated, it works now.  Pls review it.

Reply:
I've seen it's done in server.js but hasn't changed in User.js or Image.js

Reply:
just updated now.